docs/hub/webhooks.md

```diff
@@ -0,0 +1,120 @@
+# Webhooks
+
+Webhooks allow you to know when new changes happen on specific repos or repos belonging to specific users / organizations (not just your repos, but any repo!).
+
+You can configure the webhooks in your [settings](https://huggingface.co/settings/webhooks).
+
+You can watch repo changes or community events. If you watch all repos belonging to a user or an organization, you can also receive events for repo creation.
+
+## Webhook payloads
+
+After registering a webhook, you will be notified of events via an `HTTP POST` call on the specified URL. The payload is encoded in JSON.
+
+To determine the nature of the event, you can look at the `event` and `domain` fields in the payload.
+
+You can check the last payloads sent in the activity tab of the webhook page, as well as replay webhooks.
+
+### Event & Domain
+
+The top-level properties `event` and `domain` are always specified and used to determine the nature of the event.
+
+`event` can take one of the following values: 
+
+- `"create"`
```

Comment:
What about "duplicated" ? Guess that only exists for spaces yet no?

Reply:
It's a `"create"` event